packages/cookbook/package.json

```diff
@@ -1,16 +1,14 @@
 {
-  "name": "cookbook",
+  "name": "@near-js/cookbook",
+  "private": true,
   "version": "1.0.0",
   "description": "",
   "main": "main.js",
-  "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
-  },
   "author": "",
   "license": "ISC",
   "dependencies": {
     "chalk": "^4.1.1",
     "homedir": "^0.6.0",
-    "near-api-js": "file:../../"
+    "near-api-js": "workspace:*"
```

Comment:
This indicates to `pnpm` that it should create a symlink to the `near-api-js` package within the workspace.